hashed-locations.md

```diff
@@ -93,6 +93,31 @@ module SOLIDITY-FIELDS
     syntax Int ::= #hash ( Int , Int ) [function, klabel(contract_access_hash), symbol]
  // -----------------------------------------------------------------------------------
     rule #hash(I1, I2) => keccak(#bufStrict(32, I2) ++ #bufStrict(32, I1))
+endmodule
+```
+
+### Foundry Success Predicate
+
+Foundry has several baked-in convenience accounts for helping to define the "cheat-codes".
+Here we define their addresses, and important storage-locations.
+
+```k
+module FOUNDRY-ACCOUNTS
+    imports SOLIDITY-FIELDS
+
+    syntax Contract        ::= FoundryContract
+    syntax Field           ::= FoundryField
+    syntax FoundryContract ::= "Foundry"      [klabel(contract_Foundry)]
+                             | "FoundryTest"  [klabel(contract_FoundryTest)]
+                             | "FoundryCheat" [klabel(contract_FoundryCheat)]
+ // -------------------------------------------------------------------------
+    rule #address(Foundry)      => 137122462167341575662000267002353578582749290296  // 0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38
```

Comment:
Why is this moved here? It should be in `foundry.md`.

If the problem is having `Contract` sort defined, only move that definition.

Reply:
Same for `Field` sort.

Reply:
```
syntax Contract
syntax Field
```